pkgs/tools/networking/xrootd/default.nix

```diff
@@ -16,6 +16,9 @@
 , voms
 , zlib
 , enableTests ? true
+  # If not null, the builder will
+  # move "$out/etc" to "$out/etc.orig" and symlink "$out/etc" to externalEtc.
+, externalEtc ? "/etc"
```

Comment:
This non-null default is setup to work on non-nixos, correct?

Reply:
Well, it would be reasonable (only) with non-NixOS and NixOS-with-`environment.etc` setup. After all, it also requires `/etc/grid-security` to get data from CERN GRID.

Reply:
Then let's drop the option if we don't expect to exercise it.

Reply:
There's still a use case when top-level directories are inaccessible, or when an unprivileged user wants to use an alternative `/etc` content.